src/MLJ.jl

```diff
@@ -47,40 +34,40 @@ function MMI.update(
     while cache[:info][:nrounds] < model.nrounds
       grow_evotree!(fitresult, cache, model)
     end
-    report = (features=A.names,)
+    report = (features=Tables.schema(A).names,)
```

Comment:
My understanding is that tables are not required to support `schema` directly. From the `Tables.Schema` docstring:

```
To get a table's schema, one can call Tables.schema on the result of Tables.rows or 
Tables.columns, but also note that a table may return nothing, indicating that its column
 names and/or column element types are unknown (usually not inferable). 
```

In the `fit` you are okay, because you first do `A -> Tables.columntable(A)`, but you didn't do that here. Maybe pass the names in `cache`, as they shouldn't change between `fit` and `update`.

Reply:
Done, using the `fnames` stored in `cache`.